issuer-front-end/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# This dockerfile is meant to be run from the **root of the repository**.
+
+ARG build_image=node:16-slim
+
+FROM ${build_image} AS frontend
+
+RUN yarn set version 3.2.0
```

Comment:
As long as the yarn runtime is included in the repository (as recommended), this should not be necessary.